packages/terra-clinical-item-view/src/ItemView.jsx

```diff
@@ -48,24 +56,30 @@ const defaultProps = {
   isTruncated: false,
   accessoryAlignment: 'alignCenter',
   startAccessory: undefined,
+  reserveStartAccessorySpace: false,
   endAccessory: undefined,
+  reserveEndAccessorySpace: false,
   displays: [],
   comment: undefined,
 };
 
 class ItemView extends React.Component {
 
-  static renderAccessory(accessory, accessoryAlignment) {
+  static renderAccessory(accessory, reserveSpace, accessoryAlignment, type) {
     let accessorySection;
-    if (accessory) {
+    if (accessory || reserveSpace) {
+      const reserveSpaceClassName = reserveSpace ? `reserve-${type}-accessory-space` : null;
+
       const accessoryClassNames = cx(
         'accessory',
+        `${type}-accessory`,
+        `${reserveSpaceClassName}`,
```

Comment:
You should be able to just use `reserveSpaceClassName` without the template string.